tests/countries/test_indonesia.py

```
@@ -44,6 +45,13 @@ def test_special(self):
         self.assertHoliday(dts, obs_dts)
         self.assertNoNonObservedHoliday(obs_dts)
 
+    # Test that Indonesia uses the MABIMS lunar calendar for Islamic holidays
+    def test_indonesia_uses_mabims_calendar(self):
+        id = Indonesia()
```

Ruff (0.15.10)

[error] Variable `id` is shadowing a Python builtin

(A001)